lib/LedgerSMB/Scripts/recon.pm

```diff
@@ -204,15 +204,15 @@ sub _display_report {
     $recon->unapproved_checks;
 
     my $contents = '';
-    {
+    if ($request->{csv_file}) {
```

Comment:
Are you sure this field is even ever defined? Did you try uploading a csv before submitting this PR?

I'm asking because I think it won't be defined, even when you submit a csv.